004-ipfs-in-a-container/src/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM phusion/baseimage:0.9.16
```

Comment:
The Phusion guys have a slightly different opinion on how the base images are supposed to be setup. Mainly, they have written about how process ID 1 is treated differently, since it is expected to be `init`. So the base image provides a small `init` system to run daemons and such. It's what I've been using for my Rails apps.

I'll add a comment in here pointing to the docs explaining this.

Another option is to use Vendan's slimmed down busybox image. 4MB is pretty compelling.